app/editor/src/components/toggle-group/ToggleGroup.tsx

```diff
@@ -5,14 +5,15 @@ import { Col, IOptionItem, Row, Show } from 'tno-core';
 import * as styled from './styled';
 
 interface IToggleOption {
+  id?: string | number;
```

Comment:
Resolved a number of implementation issues by adding a way to include an `id` for toggle options.  It defaults to the `label` if needed.